server/src/resolvers/getVariantsResolver/getVariantsResolver.ts

```diff
@@ -65,19 +66,45 @@ const resolveVariantQuery = async (args: QueryInput): Promise<CombinedVariantQue
     }
   });
 
-  // once variants are merged, handle annotations
-  const caddAannotations = settled.find(
+  // filter data that are not in user requested assemblyId
```

Comment:
Can we refactor the code for liftover to a separate file? This `getVariantsResolver` file is getting large.

Reply:
Are you referring to refactor code for "caddAnnotations"? There is a separate file for liftover already